docs/translate-usage.rst

```diff
@@ -0,0 +1,109 @@
+Using the API
+=============
+
+With `Google Translate`_, you can dynamically translate text
+between thousands of language pairs. The Google Translate API
+lets websites and programs integrate with Google Translate
+programmatically. Google Translate API is available as a
+paid service. See the `Pricing`_ and `FAQ`_ pages for details.
+
+Authentication / Configuration
+------------------------------
+
+- Use :class:`~gcloud.translate.client.Client` objects to configure
+  your applications.
+
+- :class:`~gcloud.translate.client.Client` objects hold both a ``key``
+  and a connection to the Translate service.
+
+- **An API key is required for Translate.** See
+  `Identifying your application to Google`_ for details. This is
+  significantly different than the other clients in ``gcloud-python``.
+
+Methods
+-------
+
+To create a client:
+
+  .. code::
+
+     >>> from gcloud import translate
+     >>> client = translate.Client('my-api-key')
+
+The API has three supported methods:
+:meth:`~gcloud.translate.client.Client.get_languages`,
+:meth:`~gcloud.translate.client.Client.detect_language` and
+:meth:`~gcloud.translate.client.Client.translate`.
+
+To get a list of languages supported by Google Translate
+
+  .. code::
+
+     >>> from gcloud import translate
+     >>> client = translate.Client('my-api-key')
+     >>> client.get_languages('en')
```

Comment:
Yea, this is a bit confusing at first. It's basically saying "if you want the name of the language, tell us what language you want it in...

Check the bottom of https://googlecloudplatform.github.io/gcloud-ruby/#/docs/v0.8.2/gcloud/translate 

> To receive the names of the supported languages, as well as their ISO 639-1 codes, provide the code for the language in which you wish to receive the names.

---- 

And obligatory meme...

![we-need-to-go-deeper](https://cloud.githubusercontent.com/assets/112928/15150163/1a55e404-169a-11e6-9691-d12f6015d0d6.jpg)